Hymns/Display/DisplayHymnView.swift

```diff
@@ -22,7 +22,10 @@ struct DisplayHymnView: View {
                 Text(viewModel.title)
                     .fontWeight(.semibold)
                 Spacer()
-                Image(systemName: "heart").padding()
+                Button(action: {self.viewModel.toggleFavorited()
+                }, label: {
+                    self.viewModel.favoritedStatus ? Image(systemName: "heart.fill").accentColor(Color.blue) : Image(systemName: "heart").accentColor(Color.black)
```

Comment:
Can you use accent color and primary color instead? That way if we ever want to change the coloring, we can just do a simple app-wide change by changing one value

Reply:
Okay, I made a color extension for this in the common folder. Is that what you had in mind.

Reply:
no, there's already an .accent color and .primarycolor that you can use directly